### PR TITLE
Added vel ang temp publisher

### PR DESCRIPTION
**Pull request by VHolonec**
Changed adi imu publisher to vel ang temp publisher

### CODE REVIEW

File: src/velangtemp_ros_publisher.cpp [hunk 1]
```diff
@@ -18,71 +18,56 @@
...
-            RCLCPP_INFO(rclcpp::get_logger("rclcpp_adiimu"), "Publishing adi imu delta angle data: '%f' '%f' '%f'",
+            RCLCPP_INFO(rclcpp::get_logger("rclcpp_velangtemp"), "Publishing delta angle data: '%f' '%f' '%f'",
                         m_message.delta_angle.x, m_message.delta_angle.y, m_message.delta_angle.z);
-            RCLCPP_INFO(rclcpp::get_logger("rclcpp_adiimu"), "Publishing adi imu temperature data: '%f' ",
+            RCLCPP_INFO(rclcpp::get_logger("rclcpp_velangtemp"), "Publishing temperature data: '%f' ",
                         m_message.temp);
-            RCLCPP_INFO(rclcpp::get_logger("rclcpp_adiimu"), "Publishing adi imu time %d us ",
-                        std::chrono::duration_cast<std::chrono::microseconds>(done-started).count());
 
             if(success)
```
Comment (rbudai98) [inline src/velangtemp_ros_publisher.cpp:72]: This is just a small suggestion, but maybe checking the success variable might be useful before logging the message (and moving the Logging lines within the if condition), this way it won't log random values

File: src/velangtemp_data_provider.cpp [hunk 2]
```diff
@@ -46,14 +46,6 @@ imu_ros2::msg::AdiImuData AdiImuDataProvider::getData(bool& success)
 
     if(success)
     {
```
Comment (rbolboac) [inline src/velangtemp_data_provider.cpp:48]: this is related to how the device is working: for burst_data_selection 1 you will have available delta angle and delta velocity data when reading from the buffer, and for burst_data_selection 0 you will have accelerometer and gyroscope data when reading from the buffer. this configuration has to be added in the code before retrieving the data.

Comment (rbolboac) [reply src/velangtemp_data_provider.cpp:48]: As discussed, this will be implemented in another PR. The current PR looks good to me.

**Review by rbolboac — APPROVED** [review]

File: src/velangtemp_ros_publisher.cpp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            if(success)
+            {
+                RCLCPP_INFO(rclcpp::get_logger("rclcpp_velangtemp"), "Publishing delta velocity data: '%f' '%f' '%f'",
+                            m_message.delta_vel.x, m_message.delta_vel.y, m_message.delta_vel.z);
+                RCLCPP_INFO(rclcpp::get_logger("rclcpp_velangtemp"), "Publishing delta angle data: '%f' '%f' '%f'",
+                            m_message.delta_angle.x, m_message.delta_angle.y, m_message.delta_angle.z);
+                RCLCPP_INFO(rclcpp::get_logger("rclcpp_velangtemp"), "Publishing temperature data: '%f' ",
+                            m_message.temp);
+
+                if(success)
```
Comment (rbudai98) [inline src/velangtemp_ros_publisher.cpp:74]: this might not need extra check

**Review by rbudai98 — APPROVED** [review]

**Review by SeptimiuVana — APPROVED** [review]

**Pull request merged**